docs/quickstart-guide.md

```diff
@@ -0,0 +1,169 @@
+# Quickstart Guide
+
+_This Quickstart Guide is written with Rails 5 in mind._
+
+## Installation
+
+1) Add the gem to your `Gemfile` and run `bundle install`.
+
+```ruby
+gem 'fae-rails'
+```
+
+2) Run Fae's installer.
+
+```bash
+rails g fae:install
+```
+
+3) Start your Rails server and visit http://localhost:3000/admin to setup your super admin user.
+
+That's it, you are up and running with Fae!
+
+# An Example CMS
+
+To give you an idea of where to go from there, here is a brief walkthrough of an example CMS setup. This example CMS will manage articles, the article categories they belong to and content on an about page.
+
+## Fae's Scaffold Generator
+
+To add an object to manage in Fae you can use Fae's built in generators to create all necessary files. They are built on the Rails generators and work very similarly.
+
+**First let's scaffold our `ArticleCategory` object:**
+
+```bash
+rails g fae:scaffold ArticleCategory name:string position:integer
+```
+
+**Next is our `Article` object:**
+
+```bash
+rails g fae:scaffold Article title:string slug:string introduction:text body:text date:date hero_image:image pdf:file article_category:references
+```
+
+This one has a couple special Fae moves in it. `image` and `file` are custom attribute types that setup everything Fae needs to manage those assets.
+
+_[Learn more about managing images and files](https://www.faecms.com/documentation/tutorials-image_and_files)_
+
+## Fae's Page Generator
+
+Fae has a built in pages module that let's you define content blocks on a page as attributes on a model. This allows you to get content for an about page via `AboutUsPage.instance` instead of dealing with hard-coded IDs.
+
+To use this module, you'll have to follow Fae's convention, but we have a custom genereator to get you started.
+
+**To generate content blocks for an `AboutUsPage`:**
+
+```bash
+rails g fae:page AboutUs hero_image:image headline:string body:text
+```
+
+_[Learn more about content blocks and pages](https://www.faecms.com/documentation/topics-pages)_
+
+**Finally let's add the scaffolded objects to the database:**
+
+```bash
+rails db:migrate
```

Comment:
Rails 5 adds all rake tasks to the `rails` command so you aren't jumping between both.